hil-test/.cargo/config.toml

```diff
@@ -0,0 +1,22 @@
+[env]
+DEFMT_LOG="info"
+RUST_LOG = "warn"
```

Comment:
What's the reason for setting different log levels for the two environment variables here?

Reply:
`probe-rs `was too verbose, but I'll just remove them